test/python/test_ttir_ops.py

```diff
@@ -0,0 +1,112 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# RUN: pytest -svv %s | FileCheck %s
+
+from test_infra.ttir import Add, Multiply, Exp
+from test_infra.ttir_builder import TTIRBuilder
+from ttmlir.ir import *
+
+import torch
+import pytest
+
+# ----- Fixtures -----
+
+
+@pytest.fixture
+def context() -> Context:
+    return Context()
+
+
+@pytest.fixture
+def location(context: Context) -> Location:
+    return Location.unknown(context)
+
+
+@pytest.fixture
+def builder(context: Context, location: Location) -> TTIRBuilder:
+    return TTIRBuilder(context, location)
+
+
+@pytest.fixture
+def add(context: Context, location: Location) -> Add:
+    return Add(context, location)
+
+
+@pytest.fixture
+def multiply(context: Context, location: Location) -> Multiply:
+    return Multiply(context, location)
+
+
+@pytest.fixture
+def exp(context: Context, location: Location) -> Exp:
+    return Exp(context, location)
```

Comment:
I don't think we should make the ops fixtures, we should write tests that instantiate ops directly.

Reply:
Rewritten as regular python functions.